service/controller/resource/instance/deployment.go

```diff
@@ -38,6 +38,16 @@ func (r Resource) newDeployment(ctx context.Context, obj providerv1alpha1.AzureC
 		workerBlobName,
 	}
 
+	var distroVersion string
+	{
+		for _, component := range cc.Release.Components {
+			if component.Name == key.ContainerLinuxComponentName {
+				distroVersion = component.Version
```

Comment:
Getting the version out of the components.

Reply:
Why not call it osversion?

Reply:
@stone-z called it the same in `kvm-operator ` AFAIK - so I wanted to be consistent. Maybe we change it everywhere?
I have no strong opinion here.

Reply:
If that is ok with you, sure :) I just wanted to avoid discrepancies between OS and Distro